gatsby-ssr.js

```diff
@@ -24,9 +24,10 @@ exports.onRenderBody = function (_ref, pluginOptions) {
     manualLoad = pluginOptions.manualLoad,
     loadType = pluginOptions.loadType,
     useNewSDK = pluginOptions.useNewSDK,
-    loadOptions = pluginOptions.loadOptions;
+    loadOptions = pluginOptions.loadOptions,
+    sdkURL = pluginOptions.sdkURL;
 
-  var sdkSrc = "https://cdn.rudderlabs.com/v1/rudder-analytics.min.js";
+  var sdkSrc = sdkURL || "https://cdn.rudderlabs.com/v1/rudder-analytics.min.js";
   if (useNewSDK) {
     sdkSrc = "https://cdn.rudderlabs.com/v1.1/rudder-analytics.min.js";
   }
```

Comment:
For v1.1 SDK url is static. We need to modify the logic so that we can use the v1.1 beta

Reply:
We'll use the `sdkURL` option to supply the CDN URL of either of the beta versions. Moreover, as `v1` is deprecated, we don't need to support it here anymore.